client/app/scripts/superdesk/editor/editor.js

```diff
@@ -556,11 +556,14 @@ angular.module('superdesk.editor', ['superdesk.editor.spellcheck'])
 
     .directive('sdTextEditor', ['editor', 'spellcheck', '$timeout', function (editor, spellcheck, $timeout) {
 
+        var disableToolbar = window.config.server.disableEditorToolbar || false;
```

Comment:
here it should use `config` value via angular di - so `function(editor, spellcheck, $timeout, config) { ...`